packages/cloud_functions/cloud_functions/test/firebase_functions_test.dart

```diff
@@ -2,74 +2,156 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// @dart=2.9
-
-import 'package:firebase_core/firebase_core.dart';
 import 'package:cloud_functions/cloud_functions.dart';
 import 'package:cloud_functions_platform_interface/cloud_functions_platform_interface.dart';
+import 'package:firebase_core/firebase_core.dart';
 import 'package:flutter_test/flutter_test.dart';
 
 import 'mock.dart';
 
-import 'package:mockito/mockito.dart';
-
 void main() {
-  setupFirebaseFunctionsMocks();
-  FirebaseFunctions functions;
-  FirebaseFunctions functionsSecondary;
-  FirebaseApp app;
-  FirebaseApp secondaryApp;
-
-  group('$FirebaseFunctions', () {
-    setUpAll(() async {
-      app = await Firebase.initializeApp();
-      kMockFirebaseFunctionsPlatform =
-          MockFirebaseFunctionsPlatform(app, 'test_region');
-      FirebaseFunctionsPlatform.instance = kMockFirebaseFunctionsPlatform;
-
-      secondaryApp = await Firebase.initializeApp(
+  setupFirebaseCoreMocks();
+
+  setUpAll(() async {
```

Comment:
why `setupAll` instead of `setup`?
To avoid reusing the `MockFirebaseFunctionsPlatform` between tests

Reply:
Done